docs/book/src/developer/logging.md

```diff
@@ -0,0 +1,158 @@
+# Logging
+The Cluster API project is committed to improving the SRE/developer experience when troubleshooting issues, and logging
+plays an important part in this goal.
+
+In Cluster API we strive to follow three principles while implementing logging:
+
+- **Logs are for SRE & developers, not for users!**
```

Comment:
We should define `users` in this context, SRE & developers can be users at the same time

Reply:
Would you want to say application developer? Who's this persona?

Reply:
Switched to end user, defined in https://cluster-api.sigs.k8s.io/user/personas.html#service-provider-end-userconsumer